CHANGELOG.unreleased.md

```diff
@@ -29,4 +29,6 @@
 
 <!--packages-start-->
 
+- xo-server-transport-slack patch
```

Comment:
Please add a CHANGELOG entry because this issue has been seen (reported) by users :slightly_smiling_face: